docs/content/docs/deployment/config.md

```diff
@@ -27,16 +27,100 @@ under the License.
 
 # Configuration
 
-All configuration is done in `conf/flink-conf.yaml`, which is expected to be a flat collection of [YAML key value pairs](http://www.yaml.org/spec/1.2/spec.html) with format `key: value`.
+All configuration is done in Flink Configuration File (see [Flink Configuration File](#flink-configuration-file)).
```

Comment:
> All configuration is done in Flink Configuration File

-> All configuration can be set in Flink configuration file in the `conf/` directory.